src/modules/healtcheck/healtcheck.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable} from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { MarketEntity } from '../markets/entities/market.entity';
+import { UkefNumber as Number } from '../numbers/entities/number.entity';
+
+@Injectable()
+export class HealtcheckService {
+
+  constructor(
+    @InjectRepository(Number, "mssql-number-generator")
+      private readonly numberRepository: Repository<Number>,
+    @InjectRepository(MarketEntity, "mssql-cis")
+      private readonly marketsRepository: Repository<MarketEntity>,
+    ) {}
+
+  async checkAllDatabases(): Promise<any> {
+    let checks = {
```

Comment:
we should use `const` if the variable is not being re-assigned again

Reply:
Done, eslint fixed this.